app/views/ReportView/components/GenomicSummary/index.tsx

```diff
@@ -331,16 +333,20 @@ const GenomicSummary = ({
         `/reports/${report.ident}/summary/genomic-alterations-identified/${chipIdent}`,
         { comment },
       );
-      await req.request(isSigned);
 
+      if (isSigned) {
+        showConfirmDialog(req);
+      } else {
+        await req.request();
+      }
       setVariantCounts((prevVal) => ({ ...prevVal, [type]: prevVal[type] - 1 }));
```

Comment:
whoops I think we'll need to put all the 

```
setVariantCounts((prevVal) => ({ ...prevVal, [type]: prevVal[type] - 1 }));
setVariants((prevVal) => (prevVal.filter((val) => val.ident !== chipIdent)));
snackbar.success('Entry deleted');
```

under the else condition due to javascript scoping problems